src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -503,6 +504,28 @@ void describeChannelBundles(ColibriConferenceIQ iq)
     }
 
     /**
+     * Adds the endpoint of this <tt>Conference</tt> as
+     * <tt>ColibriConferenceIQ.Endpoint</tt> instances in <tt>iq</tt>.
+     * @param iq the <tt>ColibriConferenceIQ</tt> in which to describe.
+     */
+    void describeEndpoints(ColibriConferenceIQ iq)
+    {
+        synchronized (transportManagers)
+        {
+            for (Map.Entry<String, IceUdpTransportManager> entry
+                : transportManagers.entrySet())
```

Comment:
Why do we iterate over the transport managers? There might be endpoints that don't use channel bundles. Can we iterate over the endpoints instead?